CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ endif()
 
 project(BayesFilters
         LANGUAGES CXX
-        VERSION 0.8.101)
+        VERSION 0.9.101)
```

Comment:
There is no need to update the minor version to `9`. This is the `devel` branch of the next version that will have minor version `8`. All the breaking changes will go under that number and it is compliant with our custom devel SemVer.

Reply:
Done and also changed 101 to 100 as discussed.